1_crosswalk_fetch/src/fetch_wqp_sites.R

```diff
@@ -1,5 +1,7 @@
 fetch_wqp_sites <- function(out_ind, characteristicName, sf_ind, box_res, dummy, ...) {
 
+  sf::sf_use_s2(FALSE)
+  on.exit(sf::sf_use_s2(TRUE))
```

Comment:
what is s2? I am reading the function help but it is not very illuminating.

Reply:
Good point! Will add comments